renovate.json

```diff
@@ -8,7 +8,7 @@
   ],
   "postUpdateOptions": ["yarnDedupeHighest"],
   "labels": ["dependencies"],
-  "separateMinorPatch": true,
+  "separateMinorPatch": false,
```

Comment:
Could it just be removed? I think the default is false?

Reply:
yeah it can be remove https://docs.renovatebot.com/configuration-options/#separateminorpatch